modules/op-pipelines-as-code-splitting-logs-by-namespace.adoc

```diff
@@ -0,0 +1,14 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-pipelines-as-code.adoc
+
+:_content-type: REFERENCE
+[id="pipelines-as-code-splitting-logs-by-namespace_{context}"]
+= {pac} splitting logs by namespace
+
+The logs contain the namespace information to make it possible to grep logs on namespace or split the logs by a particular namespace. For example, to view the logs and search for `namespace` information, enter the following command:
+
+[source,terminal]
+----
+$ oc logs pipelines-as-code-controller-59c9d5db68-fkp7n -n pipelines-as-code | grep namespace
```

Comment:
`-n openshift-pipelines` right? As this is the downstream documentation.
@savitaashture @piyush-garg

Reply:
true it will be in `openshift-pipelines` namespace

Reply:
Hello @piyush-garg @VeereshAradhya because the `pipelines-as-code` namespace is wrongly used throughout the Pipelines as Code section in our docs, I will create a separate Jira to address the issue. In this chapter, I corrected it as advised. Thank you!

Reply:
Sure that works